spec/issues/2001_jtd_only_optional_properties.spec.ts

```diff
@@ -0,0 +1,28 @@
+import _Ajv from "../ajv_jtd"
+import * as assert from "assert"
+
+const PROP_COUNT = 3
+
+describe("schema with only optional properties", () => {
+  let ajv: InstanceType<typeof _Ajv>
```

Comment:
```suggestion
  let ajv: _Ajv
```

not 100% certain, but isn't it the same?

Reply:
TSError: ⨯ Unable to compile TypeScript:
spec/issues/2001_jtd_only_optional_properties.spec.ts:8:12 - error TS2749: '_Ajv' refers to a value, but is being used as a type here. Did you mean 'typeof _Ajv'?

8   let ajv: _Ajv

Reply:
alright, ts is weird... :)

Reply:
I'm not a TypeScript expert, so I had to Google a solution, hope it's good